docs/mine/lotus/miner-configuration.md

```diff
@@ -371,15 +371,19 @@ The fees section allows to set limits to the gas consumption for the different m
   MaxPublishDealsFee = "0.05 FIL"
   MaxMarketBalanceAddFee = "0.007 FIL"
   [Fees.MaxPreCommitBatchGasFee]
-      Base = "0.025 FIL"
-      PerSector = "0.025 FIL"
+      Base = "0 FIL"
+      PerSector = "0.02 FIL"
   [Fees.MaxCommitBatchGasFee]
-      Base = "0.05 FIL"
-      PerSector = "0.05 FIL"
+      Base = "0 FIL"
+      PerSector = "0.03 FIL" 
 
 ```
 
-Depending on the network congestion, the base fee for a transaction may grow or decrease. Your gas limits will have to be larger than the base fee for the messages to be included. A very large max fee can, however, result in the quick burning of funds when the base fees are very high, as the miner automatically submits messages during normal operation, so be careful about this. It is also necessary to have more funds available than any max fee set, even if the actual fee will be far less than the max fee set. \*MaxWindowPostGasFee is currently reduced, but the setting used should remain fairly high, e.g., 2 FIL.
+Depending on the network congestion, the base fee for a transaction may grow or decrease. Your gas limits will have to be larger than the base fee for the messages to be included. A very large max fee can, however, result in the quick burning of funds when the base fees are very high, as the miner automatically submits messages during normal operation, so be careful about this. It is also necessary to have more funds available than any max fee set, even if the actual fee will be far less than the max fee set.
+
+Set the maximum cost you are willing to pay for onboarding **per** sector in `MaxPreCommitBatchGasFee.PerSector`/`MaxCommitBatchGasFee.PerSector` to avoid unexpected high cost. 
```

Comment:
```suggestion
Set the maximum cost you are willing to pay for onboarding **per** sector in `MaxPreCommitBatchGasFee.PerSector`/`MaxCommitBatchGasFee.PerSector` to avoid unexpected high costs. 
```